### ISSUE TITLE
Dota coach not joining voice channel | public audio disabled

### ISSUE THREAD

**Issue by gerdeuc** [cannawen/dota-gsi-discord-bot#149]
Hi ma'am can i ask how to fix this? there is no bot in voice channel in our server when we /coachme the bot not joining in our voice channel and when we click the link its says that 'public audio disabled'. I try to do step 1 in adding dota coach but its still the same. I try it in new server, it works. But in our server for dota it doesnt. Thank you

**Comment by cannawen** [comment]
Does the bot have permissions to access to the voice channel that the users are in?

**Comment by gerdeuc** [comment]
> Does the bot have permissions to access to the voice channel that the users are in?

Yes ma’am. The bot has permission. But its still not joining. No one in out channel is usong the bot. But when we /coachme its not joining and always public audio disabled. I try to invite again the bot but still the same. I try it in new server and it works fine. But in our server its not working properly
![IMG_6935](https://github.com/cannawen/dota-gsi-discord-bot/assets/165653721/a3ffdb66-47bd-4e3f-842f-71838bbba45d)
![IMG_6936](https://github.com/cannawen/dota-gsi-discord-bot/assets/165653721/7733e29e-0559-4538-9216-e0f72e03cbfb)
![IMG_6937](https://github.com/cannawen/dota-gsi-discord-bot/assets/165653721/6acd6229-cb37-422c-a5d4-e1d08dc01e22)

**Comment by cannawen** [comment]
I see you are trying to get it to join a private channel. It should work if dota-coach has permissions to join the channel, like such:
<img width="1042" alt="Screen Shot 2024-04-29 at 11 51 25 PM" src="https://github.com/cannawen/dota-gsi-discord-bot/assets/1088336/0c31e345-92df-4bcf-a913-fb1d33e1a369">

If it still does not work, you can try creating a new unlocked channel and see if /coachme will work there.

**Comment by gerdeuc** [comment]
> I see you are trying to get it to join a private channel. It should work if dota-coach has permissions to join the channel, like such: <img alt="Screen Shot 2024-04-29 at 11 51 25 PM" width="1042" src="https://private-user-images.githubusercontent.com/1088336/326661781-0c31e345-92df-4bcf-a913-fb1d33e1a369.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ0NDk3NTIsIm5iZiI6MTcxNDQ0OTQ1MiwicGF0aCI6Ii8xMDg4MzM2LzMyNjY2MTc4MS0wYzMxZTM0NS05MmRmLTRiY2YtYTkxMy1mYjFkMzNlMWEzNjkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MzBUMDM1NzMyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDU0MDI0MmQzMjg2NjQxNzdlYzQ5NGVlNTFmMTc1YzM2ZGQzNWYwZGRlY2I4OTBlMDA4MzgzM2JhMzU3MTRlZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.BolHIU6WUDkGBLM4pkiO-WYnNcm7NfEEvqMqPir_5k8">
> 
> If it still does not work, you can try creating a new unlocked channel and see if /coachme will work there.

I already create new public channel but still doesnt work. Bot not joining in the voice channel. Our private voice channel has permission. Can you test our server ma’am? I’ll make you admin to check our server? 

https://discord.gg/Kjm9uRww


![IMG_6940](https://github.com/cannawen/dota-gsi-discord-bot/assets/165653721/2f8b86d3-74e9-47c6-a51b-63cf4856bb56)

![IMG_6941](https://github.com/cannawen/dota-gsi-discord-bot/assets/165653721/b9802a40-5dc5-4905-a56a-e5c773af9fcc)

**Comment by cannawen** [comment]
Permissions looks fine, I'm not sure what's wrong.

Have you tried kicking the bot from the server and then adding it back again?

**Comment by gerdeuc** [comment]
> Permissions looks fine, I'm not sure what's wrong.
> 
> Have you tried kicking the bot from the server and then adding it back again?

Yes ma’am i tried kicking the bot and invite again but still the same. I dont know why dota coach not joining in voice channel. I tried in new server it works fine.

**Comment by cannawen** [comment]
I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).

**Comment by gerdeuc** [comment]
> I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).

Hello ma’am. Bot is okay its now joining in voice channel and speaking. Thank you

**Comment by gerdeuc** [comment]
> I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).

Hello ma’am. Bot is okay its now joining in voice channel and speaking. Thank you
![Uploading IMG_6954.png…]()

**Comment by gerdeuc** [comment]
> I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).

Hello ma’am. Bot is okay its now joining in voice channel and speaking. Thank you
![IMG_6954](https://github.com/cannawen/dota-gsi-discord-bot/assets/165653721/66d8e166-f39d-4fa4-b556-e30a0f1ba0cc)

**Comment by gerdeuc** [comment]
> I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).

hello ma'am sorry to disturb you, i cant use dota coach. its still having problem connecting to voice channel. 😦 only 2 members has been able to use this

**Comment by gerdeuc** [comment]
> I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).

Is it possible to reset/restart/remove our guild server to dota coach and i will invite & start the step 1 again. Still having some problem connecting dota coach in our voice channel :(

**Comment by gerdeuc** [comment]
> > I don't know why it's not working, it must be some sort of permissions thing on that server specifically (if it is working for you on other servers but not that specific one).
> 
> hello ma'am sorry to disturb you, i cant use dota coach. its still having problem connecting to voice channel. 😦 only 2 members has been able to use this

Still having a problem everyday. I think theres a bug but on dota coach in our server. Sometimes its working sometimes its not. Btw thank you for your response.
